### ISSUE TITLE
Espanso commands don't work in Bash on Fresh Install (Windows 10)

### ISSUE THREAD

**Issue by kaylamillerdev** [espanso/espanso#1544]
**Describe the bug**
When trying any espanso commands in bash, it gives back the error that espanso: command not found

**To Reproduce**
1. Install espanso witth add to path checked
2. open a bash terminal
3. type in espanso status or any other espanso command

**Expected behavior**
I espected espanso status to return a message saying that it is running

**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.
![image](https://user-images.githubusercontent.com/52870645/224071372-1e9897ea-49d3-40d2-b6ed-2ae910689766.png)

**Logs**
If possible, run `espanso log` in a terminal after the bug has occurred, then post the output here so that we can better diagnose the problem

**Setup information**
- OS: What OS are you using? Windows 10
- Version: which version of Espanso are you running? (you can find out by running `espanso --version` inside a terminal) Version 2 (Latest version)

**Comment by kaylamillerdev** [comment]
I probably need to add a line to my bash prifile file but i can't figure out what that line is

**Comment by jwigg-netscout** [comment]
Yeah, I'm not positive, but I think this would be expected behavior because of the combination of

- the difference between how Windows and Linux handle adding apps to the PATH in general
- the way that WSL handles the PATH variable across the Windows<->WSL boundary. 

There's a [StackOverflow thread](https://stackoverflow.com/questions/38920710/how-can-i-run-a-windows-executable-from-wsl-ubuntu-bash) that discusses various ways to call Windows apps from within WSL, though to me, each of them looks ...not great.

of the available options, the easiest one would be to just add an alias to your bashrc that uses the full path of espanso on the Windows side of the boundary, eg `/mnt/c/Program Files/espanso/espanso.exe` (I'm not on Windows, so I have no idea if that's the correct path). However, that's only the easiest thing to do. The least awful thing I could come up with is something akin to the following (I don't have a Windows box handy to test with, so this may need some tweaking before it works):

```bash
function win() {
powershell.exe Start $@
}
```
which would (theoretically) let you use `win` as a prefix for any app in the Windows environment, so (theoretically) `win espanso status` would work as expected. 🤞

**Comment by AucaCoyan** [comment]
I have no clue why, but for some reason in my machine installed the CLI `espansod`, and it works both in bash and powershell. Can you try opening a powershell console and write `espansod` or `espanso` and press tab to autocomplete?

**Comment by kpym** [comment]
chack your path : 
```bash
> echo $PATH
```
or to check if one of the paths has 'espanso' inside you can do
```bash
echo $PATH | tr ':' '\n' | grep 'espanso'
```
